packages/conf-bmake/conf-bmake.1.0/descr

```diff
@@ -0,0 +1,3 @@
+Virtual package relying on a BSD Make compatible program
```

Comment:
The first line here should be a synopsis and stand alone and the rest of the file should be a more complete description and also stand alone.